### PR TITLE
Feature is share (final including replica)

### PR DESCRIPTION
**Pull request by deepaksibm**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

File: website/docs/d/is_share_profile.html.markdown [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+The following arguments are supported:
+
+* `name` - (Required, string) The file share profile name.
+
+## Attribute Reference
+
+The following attributes are exported:
+
+* `id` - The unique identifier of the ShareProfile.
```
Comment (sumitkumartiwari) [inline website/docs/d/is_share_profile.html.markdown:32]: The Id is coming like a date format in datasource output is not correct. pls check..

File: website/docs/d/is_share.html.markdown [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  - `crn` - The CRN for this file share.
+  - `deleted` - If present, this property indicates the referenced resource has been deleted and providessome supplementary information. Nested `deleted` blocks have the following structure:
+    - `more_info` - Link to documentation about deleted resources.
+  - `href` - The URL for this file share.
+  - `id` - The unique identifier for this file share.
+  - `name` - The unique user-defined name for this file share.
+  - `resource_type` - The resource type.
+- `zone` - The name of the zone this file share will reside in.
+- `access_tags`  - (String) Access management tags associated to the share.
+- `tags`  - (String) User tags associated for to the share.
```
Comment (sumitkumartiwari) [inline website/docs/d/is_share.html.markdown:98]: keep all attributes in alphabetical sorting order.

File: website/docs/r/is_share.html.markdown [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+  - `more_info` - Link to documentation about this status reason.
+- `share_targets` - Mount targets for the file share. Nested `share_targets` blocks have the following structure:
+	- `deleted` - If present, this property indicates the referenced resource has been deleted and providessome supplementary information. Nested `deleted` blocks have the following structure:
+		- `more_info` - Link to documentation about deleted resources.
+	- `href` - The URL for this share target.
+	- `id` - The unique identifier for this share target.
+	- `name` - The user-defined name for this share target.
+	- `resource_type` - The type of resource referenced.
+- `access_tags`  - (String) Access management tags associated to the share.
+- `tags`  - (String) User tags associated for to the share.
```
Comment (sumitkumartiwari) [inline website/docs/r/is_share.html.markdown:117]: Attribute Reference - sorting in order - same as above

File: website/docs/r/is_share_replica_operations.html.markdown [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `share_replica` - (Required, string) The file share identifier.
+* `fallback_policy` - (Optional, string) The action to take if the failover request is accepted but cannot be performed or times out. Accepted values are **split**, **fail**
+* `timeout` - (Optional, string) The failover timeout in seconds. Required with `fallback_policy`
+* `split_share` - (Boolean, string) If set to true the replication relationship between source share and replica will be removed.
+
```
Comment (sumitkumartiwari) [inline website/docs/r/is_share_replica_operations.html.markdown:38]: Here is same sorting order is important

File: website/docs/r/is_share_target.html.markdown [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `share` - (Required, string) The file share identifier.
+* `vpc` - (Required, string) The VPC in which instances can mount the file share using this share target.This property will be removed in a future release.The `subnet` property should be used instead.
+* `name` - (Required, string) The user-defined name for this share target. Names must be unique within the share the share target resides in. If unspecified, the name will be a hyphenated list of randomly-selected words.
+* `subnet` - (Optional, string) The unique identifier of the subnet associated with this file share target.Only virtual server instances in the same VPC as this subnetwill be allowed to mount the file share.In the future, this property may be required and used to assignan IP address for the file share target.
+## Attribute Reference
```
Comment (sumitkumartiwari) [inline website/docs/r/is_share_target.html.markdown:42]: Please check all places the argument and attributes references should be in alphabetically sorting order.

File: common/github.com/deepaksibm/vpc-go-sdk-beta/common/headers.go [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+import (
+	"fmt"
+	"runtime"
+)
+
+const (
+	HEADER_NAME_USER_AGENT = "User-Agent"
+
+	SDK_NAME = "vpc-go-sdk"
```
Comment (BShrivastav) [inline common/github.com/deepaksibm/vpc-go-sdk-beta/common/headers.go:11]: Is this right ?

Comment (deepaksibm) [reply common/github.com/deepaksibm/vpc-go-sdk-beta/common/headers.go:11]: ignore changes in temporary sdk folders as these will be removed while raising upstream pr

File: website/docs/d/is_share.html.markdown [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+subcategory: "Virtual Private Cloud API"
+---
+
+# ibm\_is_share
+
+Provides a read-only data source for Share. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+## Example Usage
+
+```hcl
```
Comment (SunithaGudisagarIBM) [inline website/docs/d/is_share.html.markdown:16]: Please run terraform fmt for all the resources

File: website/docs/r/is_share.html.markdown [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+}
+```
+## Argument Reference
+
+The following arguments are supported:
+
+- `encryption_key` - (Optional, String) The CRN of the [Key Protect Root Key](https://cloud.ibm.com/docs/key-protect?topic=key-protect-getting-started-tutorial) or [Hyper Protect Crypto Service Root Key](https://cloud.ibm.com/docs/hs-crypto?topic=hs-crypto-get-started) for this resource.
+- `initial_owner_gid` - (Optional, int) The initial group identifier for the file share.
+- `initial_owner_uid` - (Optional, int) The initial user identifier for the file share.
+- `iops` - (Optional, int) The maximum input/output operation performance bandwidth per second for the file share.
```
Comment (SunithaGudisagarIBM) [inline website/docs/r/is_share.html.markdown:57]: int -> Integer , string -> String,  check at all the places,

File: website/docs/r/is_share.html.markdown [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+- `iops` - The maximum input/output operation performance bandwidth per second for the file share.
+- `resource_group` - The unique identifier for this resource group. If unspecified, the account's [default resourcegroup](https://cloud.ibm.com/apidocs/resource-manager#introduction) is used.
+- `replication_role`  - The replication role of the file share.* `none`: This share is not participating in replication.* `replica`: This share is a replication target.* `source`: This share is a replication source.
+- `replication_status` - "The replication status of the file share.* `initializing`: This share is initializing replication.* `active`: This share is actively participating in replication.* `failover_pending`: This share is performing a replication failover.* `split_pending`: This share is performing a replication split.* `none`: This share is not participating in replication.* `degraded`: This share's replication sync is degraded.* `sync_pending`: This share is performing a replication sync.
```
Comment (SunithaGudisagarIBM) [inline website/docs/r/is_share.html.markdown:108]: This is not looking good, check if we can add this in bullet

File: website/docs/r/is_share.html.markdown [hunk 10]
```diff
@@ -0,0 +1,121 @@
...
+layout: "ibm"
+page_title: "IBM : is_share"
+description: |-
+  Manages Share.
+subcategory: "Virtual Private Cloud API"
+---
+
+# ibm\_is_share
+
+Provides a resource for Share. This allows Share to be created, updated and deleted.
```
Comment (SunithaGudisagarIBM) [inline website/docs/r/is_share.html.markdown:11]: Please provide the link to documentation here

File: website/docs/r/is_share_replica_operations.html.markdown [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+layout: "ibm"
+page_title: "IBM : is_share_replica_operations"
+description: |-
+  Manages Share replica operations fallback and split.
+subcategory: "Virtual Private Cloud API"
+---
+
+# ibm\_is_share_target
+
+Provides a resource for ShareTarget. This allows ShareTarget to be created, updated and deleted.
```
Comment (SunithaGudisagarIBM) [inline website/docs/r/is_share_replica_operations.html.markdown:11]: Please provide the link to documentation here

File: website/docs/r/is_share_target.html.markdown [hunk 12]
```diff
@@ -0,0 +1,52 @@
...
+layout: "ibm"
+page_title: "IBM : is_share_target"
+description: |-
+  Manages ShareTarget.
+subcategory: "Virtual Private Cloud API"
+---
+
+# ibm\_is_share_target
+
+Provides a resource for ShareTarget. This allows ShareTarget to be created, updated and deleted.
```
Comment (SunithaGudisagarIBM) [inline website/docs/r/is_share_target.html.markdown:11]: Please provide the link to documentation here

File: website/docs/r/is_share_target.html.markdown [hunk 13]
```diff
@@ -0,0 +1,52 @@
...
+  vpc = ibm_is_vpc.vpc.id
+  name = "my-share-target"
+}`
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `share` - (Required, string) The file share identifier.
```
Comment (SunithaGudisagarIBM) [inline website/docs/r/is_share_target.html.markdown:38]: `*` to `-` as we use `-` every where

File: website/docs/d/is_share_profile.html.markdown [hunk 14]
```diff
@@ -0,0 +1,35 @@
...
+page_title: "IBM : is_share_profile"
+sidebar_current: "docs-ibm-datasource-is-share-profile"
+description: |-
+  Get information about ShareProfile
+subcategory: "Virtual Private Cloud API"
+---
+
+# ibm\_is_share_profile
+
+Provides a read-only data source for ShareProfile. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
```
Comment (SunithaGudisagarIBM) [inline website/docs/d/is_share_profile.html.markdown:12]: Please provide the link to documentation in all the resource

File: ibm/service/vpc/data_source_ibm_is_share_target.go [hunk 15]
```diff
@@ -0,0 +1,359 @@
...
+		err = d.Set("vpc", dataSourceShareTargetFlattenVpc(*shareTarget.VPC))
+		if err != nil {
+			return diag.FromErr(fmt.Errorf("Error setting vpc %s", err))
+		}
+	}
+
+	return nil
+}
+
+// func dataSourceShareTargetFlattenSubnet(result vpcbetav1.SubnetReference) (finalList []map[string]interface{}) {
```
Comment (SunithaGudisagarIBM) [inline ibm/service/vpc/data_source_ibm_is_share_target.go:266]: remove commented code if its not required anymore

File: ibm/service/vpc/data_source_ibm_is_share_target.go [hunk 16]
```diff
@@ -0,0 +1,359 @@
...
+	if subnetItem.Href != nil {
+		subnetMap["href"] = subnetItem.Href
+	}
+	if subnetItem.ID != nil {
+		subnetMap["id"] = subnetItem.ID
+	}
+	if subnetItem.Name != nil {
+		subnetMap["name"] = subnetItem.Name
+	}
+	/*
```
Comment (SunithaGudisagarIBM) [inline ibm/service/vpc/data_source_ibm_is_share_target.go:295]: remove commented code if its not required anymore

File: ibm/service/vpc/data_source_ibm_is_share_target.go [hunk 17]
```diff
@@ -0,0 +1,359 @@
...
+	if vpcItem.Href != nil {
+		vpcMap["href"] = vpcItem.Href
+	}
+	if vpcItem.ID != nil {
+		vpcMap["id"] = vpcItem.ID
+	}
+	if vpcItem.Name != nil {
+		vpcMap["name"] = vpcItem.Name
+	}
+	/*
```
Comment (SunithaGudisagarIBM) [inline ibm/service/vpc/data_source_ibm_is_share_target.go:342]: remove commented code if its not required anymore

File: ibm/service/vpc/data_source_ibm_is_share_targets.go [hunk 18]
```diff
@@ -0,0 +1,339 @@
...
+	if targetsItem.MountPath != nil {
+		targetsMap["mount_path"] = targetsItem.MountPath
+	}
+	if targetsItem.Name != nil {
+		targetsMap["name"] = targetsItem.Name
+	}
+	if targetsItem.ResourceType != nil {
+		targetsMap["resource_type"] = targetsItem.ResourceType
+	}
+	// if targetsItem.Subnet != nil {
```
Comment (SunithaGudisagarIBM) [inline ibm/service/vpc/data_source_ibm_is_share_targets.go:245]: remove commented code if its not required anymore

File: ibm/service/vpc/data_source_ibm_is_share_targets.go [hunk 19]
```diff
@@ -0,0 +1,339 @@
...
+	if subnetItem.Href != nil {
+		subnetMap["href"] = subnetItem.Href
+	}
+	if subnetItem.ID != nil {
+		subnetMap["id"] = subnetItem.ID
+	}
+	if subnetItem.Name != nil {
+		subnetMap["name"] = subnetItem.Name
+	}
+	/*
```
Comment (SunithaGudisagarIBM) [inline ibm/service/vpc/data_source_ibm_is_share_targets.go:282]: remove commented code if its not required anymore

File: ibm/service/vpc/resource_ibm_is_share_target.go [hunk 20]
```diff
@@ -0,0 +1,344 @@
...
+				Required:    true,
+				ForceNew:    true,
+				Description: "The unique identifier of the VPC in which instances can mount the file share using this share target.This property will be removed in a future release.The `subnet` property should be used instead.",
+			},
+			"name": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validate.InvokeValidator("ibm_is_share_target", "name"),
+				Description:  "The user-defined name for this share target. Names must be unique within the share the share target resides in. If unspecified, the name will be a hyphenated list of randomly-selected words.",
+			},
```
Comment (SunithaGudisagarIBM) [inline ibm/service/vpc/resource_ibm_is_share_target.go:48]: remove commented code if its not required anymore

File: ibm/service/vpc/resource_ibm_is_share_target.go [hunk 21]
```diff
@@ -0,0 +1,344 @@
...
+	createShareMountTargetOptions.SetShareID(d.Get("share").(string))
+	vpcid := d.Get("vpc").(string)
+	vpc := &vpcbetav1.VPCIdentity{
+		ID: &vpcid,
+	}
+	createShareMountTargetOptions.SetVPC(vpc)
+	if _, ok := d.GetOk("name"); ok {
+		createShareMountTargetOptions.SetName(d.Get("name").(string))
+	}
+	// if subnetIntf, ok := d.GetOk("subnet"); ok {
```
Comment (SunithaGudisagarIBM) [inline ibm/service/vpc/resource_ibm_is_share_target.go:123]: remove commented code if its not required anymore

File: ibm/service/vpc/resource_ibm_is_share_target.go [hunk 22]
```diff
@@ -0,0 +1,344 @@
...
+	d.Set("share_target", *shareTarget.ID)
+
+	if err = d.Set("vpc", *shareTarget.VPC.ID); err != nil {
+		return diag.FromErr(fmt.Errorf("Error setting name: %s", err))
+	}
+
+	if err = d.Set("name", *shareTarget.Name); err != nil {
+		return diag.FromErr(fmt.Errorf("Error setting name: %s", err))
+	}
+	// if shareTarget.Subnet != nil {
```
Comment (SunithaGudisagarIBM) [inline ibm/service/vpc/resource_ibm_is_share_target.go:181]: remove commented code if its not required anymore

**Review by SunithaGudisagarIBM — CHANGES_REQUESTED** [review]
Hi Deepak, 

Please find my review comments and can you please mark them as resolved once you incorporated them

Thank you
Sunitha

**Comment by SunithaGudisagarIBM** [comment]
Verified with https://pages.github.ibm.com/vpc/vpc-spec-artifacts/branch/master/swagger-ui.html?maturity=beta&version=initial#/Shares/get_share_profile api-spec and all fields looks good to me.
```
data "ibm_is_share_profiles" "profiles" {
}
data "ibm_is_share_profile" "profile" {
  name = "custom-iops"
}
```
Screenshot 2023-02-09 at 2 46 46 PM

File: ibm/service/vpc/resource_ibm_is_share.go [hunk 23]
```diff
@@ -0,0 +1,1077 @@
...
+	if hasChange {
+
+		sharePatch, err := sharePatchModel.AsPatch()
+
+		if err != nil {
+			log.Printf("[DEBUG] SharePatch AsPatch failed %s", err)
+			return diag.FromErr(err)
+		}
+		updateShareOptions.SetSharePatch(sharePatch)
+		_, response, err := vpcClient.UpdateShareWithContext(context, updateShareOptions)
```
Comment (SunithaGudisagarIBM) [inline ibm/service/vpc/resource_ibm_is_share.go:906]: Please add wait logic here for lifecycle_state to be stable

Comment (SunithaGudisagarIBM) [reply ibm/service/vpc/resource_ibm_is_share.go:906]: Screenshot 2023-02-09 at 3 17 05 PM

**Comment by SunithaGudisagarIBM** [comment]
```
data "ibm_is_share_targets" "targets1" {
  share = ibm_is_share.share.id
}

resource "ibm_is_share_target" "target" {
  name = "sunitha-target"
  vpc = "r006-567a5aec-d475-42cd-9407-052b1d4c167d"
  share = ibm_is_share.share.id 
}
```
Screenshot 2023-02-09 at 5 29 08 PM
Screenshot 2023-02-09 at 5 29 20 PM

**Comment by SunithaGudisagarIBM** [comment]
```
// data "ibm_is_share_profiles" "profiles" {
//}
//data "ibm_is_share_profile" "profile" {
//  name = "custom-iops"
//}
resource "ibm_is_share" "share" {
  zone    = "us-south-3"
  size    = 280
  name    = "tfp-sunitha-share"
  profile = "custom-iops"

  /* profile = "tier-3iops" */
  iops = 5000
}
/* resource "ibm_is_share" "share2" {
  zone = "us-south-2"
  size = 250
  iops = 3000
  name = "my-share2"
  profile = "custom-iops"
} */
resource "ibm_is_vpc" "testacc_vpc" {
  name = "tfp-sunitha"
}
resource "ibm_is_share" "share1" {
  zone    = "us-south-3"
  size    = 2000
  name    = "tfp-sunitha-file"
  profile = "tier-3iops"
  share_target_prototype {
    name = "target-1"
    vpc  = ibm_is_vpc.testacc_vpc.id
  }
  replica_share {
    name = "my-replica2"  
    replication_cron_spec = "0 */5 * * *"
    profile = "tier-3iops"
    zone = "us-south-1"
  } 
}
/* data "ibm_is_share_targets" "targets1" {
  share = ibm_is_share.share.id
}

resource "ibm_is_share_target" "target" {
  name = "sunitha-target"
  vpc = "r006-567a5aec-d475-42cd-9407-052b1d4c167d"
  share = ibm_is_share.share.id 
} */
/* data "ibm_is_share" "share1" {
  share = ibm_is_share.share1.id
}
data "ibm_is_share" "share" {
  share = ibm_is_share.share.id
}
data "ibm_is_shares" "share3" {
}
data "ibm_is_share_targets" "targets" {
  share = ibm_is_share.share2.id
}
data "ibm_is_share_target" "target" {
  share = ibm_is_share.share.id
  //share_target_name = "kjhakjh"
  share_target = ibm_is_share_target.target.share_target
  //share = "kjhbkug"
} */
/* resource "ibm_is_share" "replica1" {
  zone = "us-south-3"
  //size = 200
  source_share = ibm_is_share.share2.id
  name = "my-replica3"
  profile = "dp2" */
//replication_cron_spec = "0 */5 * * *"
//} 
/* resource "ibm_is_share" "replica" {
  zone = "us-south-2"
  //size = 200
  source_share = ibm_is_share.share.id
  name = "my-replica1"
  profile = "tier-3iops" */
// replication_cron_spec = "0 */6 * * *"
/* }  */
/* resource "ibm_is_share" "share4" {
  zone = "us-south-1"
  size = 220
  name = "my-share5"
  profile = "tier-3iops"
  replica_share {
    name = "my-replica2"  */
//replication_cron_spec = "0 */5 * * *"
/* profile = "tier-3iops"
    zone = "us-south-3"
  }
} */
```

**Comment by SunithaGudisagarIBM** [comment]
"ibm_is_share_replica_operations" testing is pending, as discussed with Deepak it is still under discussion.

**Comment by SunithaGudisagarIBM1** [comment]
```
// data "ibm_is_share_profiles" "profiles" {
//}
//data "ibm_is_share_profile" "profile" {
//  name = "custom-iops"
//}
/* resource "ibm_is_share" "share" {
  zone    = "us-south-3"
  size    = 280
  name    = "tfp-sunitha-share"
  profile = "custom-iops"

  /* profile = "tier-3iops" */
//iops = 5000
//} */
/* resource "ibm_is_share" "share2" {
  zone = "us-south-2"
  size = 250
  iops = 3000
  name = "my-share2"
  profile = "custom-iops"
} */
/* resource "ibm_is_vpc" "testacc_vpc" {
  name = "tfp-sunitha"
}
resource "ibm_is_share" "share1" {
  zone    = "us-south-3"
  size    = 2000
  name    = "tfp-sunitha-file"
  profile = "tier-3iops"
  share_target_prototype {
    name = "target-1"
    vpc  = ibm_is_vpc.testacc_vpc.id
  }
  replica_share {
    name = "my-replica2"  */
//replication_cron_spec = "0 */5 * * *"
//  profile = "tier-3iops"
//zone = "us-south-1"
//} 
//} 
/* data "ibm_is_share_targets" "targets1" {
  share = ibm_is_share.share.id
}

resource "ibm_is_share_target" "target" {
  name = "sunitha-target"
  vpc = "r006-567a5aec-d475-42cd-9407-052b1d4c167d"
  share = ibm_is_share.share.id 
} */
/* data "ibm_is_share" "share1" {
  share = ibm_is_share.share1.id
}
data "ibm_is_share" "share" {
  share = ibm_is_share.share.id
}
data "ibm_is_shares" "share3" {
}
data "ibm_is_share_targets" "targets" {
  share = ibm_is_share.share2.id
}
data "ibm_is_share_target" "target" {
  share = ibm_is_share.share.id
  //share_target_name = "kjhakjh"
  share_target = ibm_is_share_target.target.share_target
  //share = "kjhbkug"
} */
/* resource "ibm_is_share" "replica1" {
  zone = "us-south-3"
  //size = 200
  source_share = ibm_is_share.share2.id
  name = "my-replica3"
  profile = "dp2" */
//replication_cron_spec = "0 */5 * * *"
//} 
/* resource "ibm_is_share" "replica" {
  zone = "us-south-2"
  //size = 200
  source_share = ibm_is_share.share.id
  name = "my-replica1"
  profile = "tier-3iops" */
// replication_cron_spec = "0 */6 * * *"
/* }  */
/* resource "ibm_is_share" "share4" {
  zone = "us-south-1"
  size = 220
  name = "my-share5"
  profile = "tier-3iops"
  replica_share {
    name = "my-replica2"  */
//replication_cron_spec = "0 */5 * * *"
/* profile = "tier-3iops"
    zone = "us-south-3"
  }
} */

/* resource "ibm_is_vpc" "testacc_vpc" {
  name = "sunitha-tfp-test-vpc"
}
resource "ibm_is_vpc_default_routing_table" "test_ibm_is_vpc_default_routing_table" {
  depends_on = [ibm_is_vpc.testacc_vpc]
  /* vpc = ibm_is_vpc.testacc_vpc.id */
  /* name = "sunitha-tfp-test-vpc-reouting" */
 //accept_routes_from_resource_type = ["vpn_server"]
//} 

resource "ibm_is_vpc" "vpc" {
  name = "tfp-share-vpc"
}
resource "ibm_is_vpc" "vpc1" {
  name = "tfp-share-vpc-3"
}
resource "ibm_is_vpc" "vpc2" {
  name = "tfp-share-vpc-4"
}
resource "ibm_is_share" "share" {
  zone = "us-south-1"
  size = 200
  //iops = 200
  name = "my-share1"
  profile = "tier-5iops"
  tags        = ["share1", "share3"]
  access_tags = ["access:dev"]
}
resource "ibm_is_share" "is_share" {
  name           = "myshare"
  profile        = "tier-3iops"
  //resource_group = data.ibm_resource_group.default.id
  size           = 32000
  mount_targets {
    name = "tfp-tgone"
    vpc = ibm_is_vpc.vpc1.id
  }
  mount_targets {
    name = "tfp-tg2"
    vpc = ibm_is_vpc.vpc2.id
  }
  replica_share {
    name = "my-replica2" 
    replication_cron_spec = "0 */5 * * *"
    profile = "tier-3iops"
    zone = "us-south-3"
    mount_targets {
        name = "tfp-tg3"
        vpc = ibm_is_vpc.vpc1.id
    }
  }
  zone        = "us-south-2"
  tags        = ["tag1", "tag2"]
  //access_tags = ["accesstag1", "accesstag2"]
}

```
Screenshot 2023-03-18 at 4 54 43 PM

Screenshot 2023-03-18 at 4 55 26 PM

**Pull request closed without merging**